README.md

```diff
@@ -1,2 +1,5 @@
 # gatsby-starter-carbon
 Gatsby starter with the Carbon Design System
+
+##License
```

Comment:
Formatting is messed up a bit  here - should be a space between the hashtag and License